### ISSUE TITLE
Should 'classification=True' be 'classification=self.classification'?

### ISSUE THREAD

**Issue by jgh9094** [EpistasisLab/tpot2#150]
https://github.com/EpistasisLab/tpot2/blob/8ebc82cb3b4d6fadfa5228a7de3134a9edc59acb/tpot2/tpot_estimator/estimator.py#L661

I think the wrong argument is being passed to the classification variable?

**Comment by perib** [comment]
yes it should. Also I noticed that some selectors are duplicated in the search space templates provided in the same file. 

Also perhaps this file should be merged into [get_configspace.py?](https://github.com/EpistasisLab/tpot2/blob/8ebc82cb3b4d6fadfa5228a7de3134a9edc59acb/tpot2/config/get_configspace.py)

**Comment by jgh9094** [comment]
We should be okay by maintaining a unique set of selectors after processing all of them. That way no one selector is overrepresented when mutating things.

What files/functions should be merged into get_configspace.py?
